core/templates/pages/collection-editor-page/services/collection-editor-state.service.spec.ts

```diff
@@ -348,66 +339,48 @@ describe('Collection editor state service', function() {
 
   it('should return the last collection rights loaded as the same object',
     function() {
-      var previousCollectionRights = (
-        CollectionEditorStateService.getCollectionRights());
       var expectedCollectionRights = collectionRightsObjectFactory.create(
         fakeCollectionRightsBackendApiService.backendCollectionRightsObject);
-      expect(previousCollectionRights).not.toEqual(expectedCollectionRights);
```

Comment:
Why remove this?

Reply:
same previousCollectionRights is undefined